### ISSUE TITLE
Scroll Wheel Often Fails to Respond When Scrolling Through Text in Materialgram

### ISSUE THREAD

**Issue by ChocoFleece** [kukuruzka165/materialgram#39]
### Steps to reproduce

I'm not sure if this is a personal case, but both the Flatpak and GitHub source installations have this issue.
When opening any conversation page and using the mouse scroll wheel (regardless of scrolling up or down), there is a high probability (around 30% in my case) of encountering this issue.

### Expected behaviour

Expected Behavior: The page should scroll normally, with each scroll wheel notch causing the page to scroll a corresponding distance.

### Actual behaviour

The page sometimes fails to scroll properly, instead "accumulating" the scroll wheel notches and then scrolling a long distance all at once. For example, if I scroll up by 3 notches, the page doesn't respond for the first two scrolls, but on the third scroll, it jumps directly to the final position as if all three scrolls occurred at once.

### Operating system

Arch Linux x86_64   KWin (X11)   KDE Plasma 6.2.2

### Version of Telegram Desktop

5.6.1.1

### Installation source

Flatpak

### Logs

```shell
[2024.10.25 21:05:40] Launched version: 500600101, install beta: [FALSE], alpha: 0, debug mode: [FALSE]
[2024.10.25 21:05:40] Executable dir: /usr/bin/, name: materialgram
[2024.10.25 21:05:40] Initial working dir: /home/zhongzi26/
[2024.10.25 21:05:40] Working dir: /home/zhongzi26/.local/share/materialgram/
[2024.10.25 21:05:40] Command line: /usr/bin/materialgram -autostart -noupdate -tosettings
[2024.10.25 21:05:40] Executable path before check: /usr/bin/materialgram
[2024.10.25 21:05:40] Logs started
[2024.10.25 21:05:40] App ID: io.github.kukuruzka165.materialgram
[2024.10.25 21:05:40] Connecting local socket to 5b54c41decdc08d1929eedec71be7a38-materialgram...
[2024.10.25 21:05:40] Socket connect error 0, starting server and app...
[2024.10.25 21:05:40] Moved logging from '/home/zhongzi26/.local/share/materialgram/log_start0.txt' to '/home/zhongzi26/.local/share/materialgram/log.txt'!
[2024.10.25 21:05:40] Global devicePixelRatio: 1
[2024.10.25 21:05:40] QT_AUTO_SCREEN_SCALE_FACTOR: 0
[2024.10.25 21:05:40] QT_SCREEN_SCALE_FACTORS: HDMI-0=1.25;DP-0=1.25;DP-1=1.25;DP-2=1.25;DP-3=1.25;DP-4=1.25;DP-5=1.25;
[2024.10.25 21:05:40] Primary screen DPI: 120, Base: 86.4.
[2024.10.25 21:05:40] Computed screen scale: 140
[2024.10.25 21:05:40] DevicePixelRatio: 1
[2024.10.25 21:05:40] ScreenScale: 140
[2024.10.25 21:05:40] Icon theme: Papirus-Dark
[2024.10.25 21:05:40] Fallback icon theme: 
[2024.10.25 21:05:40] Media Info: Type Playback, default: STUDIO M 模拟立体声, list: { "STUDIO M 模拟立体声" <STUDIO M 模拟立体声>, "GA104 High Definition Audio Controller 数字立体声(HDMI)" <GA104 High Definition Audio Controller 数字立体声(HDMI)> }, full list refresh: true
[2024.10.25 21:05:40] Media Info: Type Capture, default: STUDIO M 模拟立体声, list: { "STUDIO M 模拟立体声" <STUDIO M 模拟立体声>, "NoiseTorch Microphone for STUDIO M Source" <NoiseTorch Microphone for STUDIO M Source>, "Monitor of GA104 High Definition Audio Controller 数字立体声(HDMI)" <Monitor of GA104 High Definition Audio Controller 数字立体声(HDMI)>, "Monitor of STUDIO M 模拟立体声" <Monitor of STUDIO M 模拟立体声>, "vencord-screen-share" <vencord-screen-share> }, full list refresh: true
[2024.10.25 21:05:40] Media Error: Can't sync default device for type Camera, default: , list: {  }
[2024.10.25 21:05:40] App Info: reading settings...
[2024.10.25 21:05:40] App Info: reading encrypted settings...
[2024.10.25 21:05:40] Lang Info: Loaded cached, keys: 7528
[2024.10.25 21:05:40] Lang Error: Unexpected tag (key 'lng_action_add_user:users')
[2024.10.25 21:05:40] Lang Error: Expected '}' after tag name (key 'lng_confirm_phone_about:phone')
[2024.10.25 21:05:40] Lang Info: Loaded cached, keys: 2457
[2024.10.25 21:05:40] Lang Error: Unexpected tag (key 'lng_call_box_status_group:count')
[2024.10.25 21:05:40] Lang Error: Expected '}' after tag name (key 'lng_confirm_phone_about:phone')
[2024.10.25 21:05:40] Lang Error: Unknown tag (key 'lng_manage_discussion_group_warning:visible')
[2024.10.25 21:05:40] Font: from ':/gui/fonts/Regular.ttf' loaded 'Google Sans'
[2024.10.25 21:05:40] Font: from ':/gui/fonts/Italic.ttf' loaded 'Google Sans'
[2024.10.25 21:05:40] Font: from ':/gui/fonts/Medium.ttf' loaded 'Google Sans'
[2024.10.25 21:05:40] Font: from ':/gui/fonts/Medium.ttf' loaded 'Google Sans Medium'
[2024.10.25 21:05:40] Font: from ':/gui/fonts/MediumItalic.ttf' loaded 'Google Sans'
[2024.10.25 21:05:40] Font: from ':/gui/fonts/MediumItalic.ttf' loaded 'Google Sans Medium'
[2024.10.25 21:05:40] Font: from ':/gui/fonts/Vazirmatn-UI-NL-Regular.ttf' loaded 'Vazirmatn UI NL'
[2024.10.25 21:05:40] Font: from ':/gui/fonts/Vazirmatn-UI-NL-SemiBold.ttf' loaded 'Vazirmatn UI NL'
[2024.10.25 21:05:40] Font: from ':/gui/fonts/Vazirmatn-UI-NL-SemiBold.ttf' loaded 'Vazirmatn UI NL SemiBold'
[2024.10.25 21:05:40] OpenAL Logging Level: (not set)
[2024.10.25 21:05:40] App Info: reading accounts info...
[2024.10.25 21:05:40] App Info: reading encrypted info...
[2024.10.25 21:05:40] App Info: reading map...
[2024.10.25 21:05:40] App Info: reading encrypted map...
[2024.10.25 21:05:40] App Info: reading encrypted user settings...
[2024.10.25 21:05:40] App Info: encrypted user settings read.
[2024.10.25 21:05:40] App Info: reading encrypted mtp data...
[2024.10.25 21:05:40] MTP Info: read keys, current: 4, to destroy: 0
[2024.10.25 21:05:40] Map read time: 19
[2024.10.25 21:05:40] App Info: reading encrypted mtp config...
[2024.10.25 21:05:40] Export Info: Destroy top bar by controller removal.
[2024.10.25 21:05:40] System tray available: [TRUE]
[2024.10.25 21:05:41] OpenGL: Could not create widget in a window.
[2024.10.25 21:05:41] OpenGL: [FALSE] (Window)
[2024.10.25 21:05:41] Notification daemon product name: Plasma
[2024.10.25 21:05:41] Notification daemon vendor name: KDE
[2024.10.25 21:05:41] Notification daemon version: 6.2.2
[2024.10.25 21:05:41] Notification daemon specification version: 1.2
[2024.10.25 21:05:41] Notification daemon capabilities: body, body-hyperlinks, body-markup, body-images, icon-static, actions, persistence, inline-reply, x-kde-urls, x-kde-origin-name, x-kde-display-appname, inhibitions
[2024.10.25 21:05:41] API Warning: not loaded minimal channel applied.
[2024.10.25 21:05:42] Lang Info: Up to date.
```

**Comment by ChocoFleece** [comment]
If you find that this is just my personal problem, please tell me and close this issue. Sorry, I am a Linux novice, I am not very good at finding the cause of the bug and fixing it, so I temporarily think it is a bug of materialgram, no offense. And, have a nice day.

**Comment by ChocoFleece** [comment]
The issue I mentioned has been fixed in the [v5.7.0.1](https://github.com/kukuruzka165/materialgram/releases/tag/v5.7.0.1) update.
